### PR TITLE
Glossary additions

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR adds more glossary definitions. 
- App Mode
- App Profile
- Cluster Mode
- Palette Virtual Cluster

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/990#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/17-glossary-all.md
Comment (antongisli) [inline content/docs/17-glossary-all.md:20]: the word "only" feels unnecessary to me "enabling you to ONLY focus on deploying ...", could just say "enabling you to focus on deploying ..."

File: content/docs/17-glossary-all.md
Comment (antongisli) [inline content/docs/17-glossary-all.md:23]: not only preconfigured but to bundle your own applications together with common necessary sercies, such as cache, db, etc, and make it easier for you to deploy those services.

File: content/docs/17-glossary-all.md
Comment (antongisli) [inline content/docs/17-glossary-all.md:128]: unnecessary definite articles? e.g. "collects information on metrics ..." rather than "collects the information on metrics ..."

**Review by antongisli — COMMENTED** [review]
apart from minor comments looks good

File: content/docs/17-glossary-all.md
Comment (resmaa) [inline content/docs/17-glossary-all.md:97]: If we can give a link to host clusters (host cluster definition within the glossary) here would be great.

**Review by resmaa — CHANGES_REQUESTED** [review]

File: content/docs/17-glossary-all.md
Comment (karl-cardenas-coding) [reply content/docs/17-glossary-all.md:23]: What do tiers mean in this context?

File: content/docs/17-glossary-all.md
Comment (karl-cardenas-coding) [reply content/docs/17-glossary-all.md:97]: @resmaa can you provide more context? How would you integrate host clusters into the definition?

Comment (antongisli) [reply content/docs/17-glossary-all.md:98]: A host cluster is just a regular kubernetes cluster that is used to host virtual clusters.

File: content/docs/17-glossary-all.md
Comment (karl-cardenas-coding) [reply content/docs/17-glossary-all.md:24]: @antongisli, it's a question intended to call out that we haven't explained what tier is in this context. You answered the question for Resma 😉

Comment (resmaa) [reply content/docs/17-glossary-all.md]: :)) yaa Karl, we were using **layers** for these **componets/ tiers/services**. added to app profile.. with Devx the tier word came in usage. That's what I believe. 
https://docs.spectrocloud.com/cluster-profiles#layers

File: content/docs/17-glossary-all.md
Comment (ritawatson) [inline content/docs/17-glossary-all.md:40]:
```suggestion
Cluster Mode enables you to create, deploy, and manage Kubernetes clusters and applications. In Cluster Mode, you can deploy Kubernetes clusters to public cloud providers, on-prem datacenters, and on the edge. 

```

File: content/docs/17-glossary-all.md
Comment (ritawatson) [inline content/docs/17-glossary-all.md:133]:
```suggestion
Spectro Agent bridges the information transfer between Palette SaaS and Palette Orchestrator. The Spectro Agent collects information such as metrics, workloads, and heartbeats and constantly updates to the SaaS platform for user access. In addition to this, the Spectro Agent is responsible for initiating and controlling Backup, OS-Patch, and Compliance Scan on the running cluster.

```

**Review by ritawatson — CHANGES_REQUESTED** [review]
Looks good except for a couple minor suggestions.

**Review by resmaa — APPROVED** [review]
lgtm

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/990#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/990#pullrequestreview-1196188153" title="Approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
